spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -1567,5 +1596,24 @@ def test_calltip(ipyconsole, qtbot):
     assert control.calltip_widget.isVisible()
 
 
+@flaky(max_runs=3)
+@pytest.mark.test_environment_interpreter
+def test_conda_env_activation(ipyconsole, qtbot):
+    """Test that the conda environment of the console is activated."""
```

Comment:
This should be

    Test that the conda environment associated with an external interpreter
    is activated before a kernel is created for it.

Reply:
Fixed